ng-dev/release/publish/actions.ts

```diff
@@ -605,11 +605,14 @@ export abstract class ReleaseAction {
    * @param versionBumpCommitSha Commit that bumped the version. The release tag
    *   will point to this commit.
    * @param isPrerelease Whether the new version is published as a pre-release.
+   * @param shouldShowAsLatestOnGitHub Whether the version released will represent
+   *   the "latest" version of the project. I.e. GitHub will show this version as "latest".
    */
   private async _createGithubReleaseForVersion(
     releaseNotes: ReleaseNotes,
     versionBumpCommitSha: string,
     isPrerelease: boolean,
+    shouldShowAsLatestOnGitHub: boolean,
```

Comment:
```suggestion
    useAsLatest: boolean,
```

Reply:
I had that initially but went with this to make it very clear what it's doing. "latest" itself in this context is quite an ambiguous term.

Reply:
I've renamed it to omit the `should`, but other than that- I think this term is quite good